app/features/onboarding/views/forms/_ci_bot_account.erb

```diff
@@ -37,7 +39,7 @@
       />
     </div>
 
-    <li>Enter your bot user account password:</li>
+    <li>Enter your bot user Personal Access Token:</li>
```

Comment:
For this one, it should along the lines of `Enter the password you'd like to use for the bot's account on fastlane.ci (not the bot github password)`
I'm not the best copy editor 🙃